src/controllers/remove.director.check.answers.controller.ts

```diff
@@ -23,7 +26,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
     const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
     const companyOfficer: CompanyOfficer = await getDirectorAndTerminationDate(session, transactionId, submissionId);
-    
+    const appointmentId = (await getOfficerFiling(session, transactionId, submissionId)).referenceAppointmentId as string
```

Comment:
should verify that appointmentId is found and not null/undefined

Reply:
Fixed